### ISSUE TITLE
String index out of bounds exception

### ISSUE THREAD

**Issue by TrademarkTM** [GamerKing195/Thirst#17]
## The error
```
[20:05:33] [Server thread/WARN]: [Thirst] Task #107 for Thirst v1.9.3 generated an exception
java.lang.StringIndexOutOfBoundsException: String index out of range: 28
	at java.lang.String.substring(String.java:1963) ~[?:1.8.0_131]
	at com.gamerking195.dev.thirst.util.UtilUpdater$1.run(UtilUpdater.java:64) ~[?:?]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71) ~[spigot-1.12.1.jar:git-Spigot-65e8124-963b19c]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:353) [spigot-1.12.1.jar:git-Spigot-65e8124-963b19c]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:739) [spigot-1.12.1.jar:git-Spigot-65e8124-963b19c]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot-1.12.1.jar:git-Spigot-65e8124-963b19c]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot-1.12.1.jar:git-Spigot-65e8124-963b19c]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot-1.12.1.jar:git-Spigot-65e8124-963b19c]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_131]
```

## Information about server
**Version:** 1.12.1
**Thirst version:** 1.9.4 - it shows as 1.9.3, but I've already updated and it's still telling me to update.

The thirst is also being shown in scoreboard and 

## Updater error
https://hastebin.com/bitiwesuba.vbs

Thanks once again!

**Comment by cadenkriese** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [BUG - Updater version comparison false positives.](https://trello.com/c/goosFxWg/58-bug-updater-version-comparison-false-positives)

**Comment by cadenkriese** [comment]
Should be fixed in version 1.9.5 found [here](https://www.spigotmc.org/resources/thirst.24610/update?update=183508). Thank you for the report! Please respond to this if it is not fixed in that version, rather than creating another issue. Apologies for the delay, I've just started school .-.
